Add project description to README PB-27 (#3)
makimo/name-microservice: README.md

# name-service

> This application is a name service, that allows assign map of unique user IDs 

Reviewer: ```suggestion
This application is a name service, which allows to assign a map of unique user IDs 
```

> on different services to each other. An example keymap would be as follows:

Reviewer: ```suggestion
on different services to each other. An example keymap is as follows:
```

```
"slack" -> "john.doe"
"jira" -> "johndoe@gmail.com"
```

> This microservice allows to i.a.:

Reviewer: ```suggestion
The microservice allows to:
```
